### ISSUE TITLE
Whitepace code not present in Euclidean chapter

### ISSUE THREAD

**Issue by leios** [algorithm-archivists/algorithm-archive#486]
I know this sounds like a joke because of how we implemented whitespace in the Euclidean algorithm chapter; however, if you highlight the code snippet, there is no actual whitespace there. It is available in the `code/whitespace` directory, though.

**Comment by jiegillet** [comment]
I know. `codeblock` (or whatever) eats the trailing spaces :(
I don't know what to do short of diving into `highlight.js`

**Comment by leios** [comment]
Yeah, it was just something to put out there. I don't think this is an easy fix.

**Comment by berquist** [comment]
It may be an easy fix, try setting this to false: https://github.com/algorithm-archivists/algorithm-archive/blob/822665537b20f41ef424baee16cbd58fc1cbcae1/book.json#L23

On Mon, Oct 8, 2018 at 8:12 AM James Schloss <notifications@github.com<mailto:notifications@github.com>> wrote:

Yeah, it was just something to put out there. I don't think this is an easy fix.

**Comment by leios** [comment]
I just tried this and it doesn't seem to work. Not sure where the whitespace is being stripped.

**Comment by Amaras** [comment]
If we're not finding a way to solve this, we might as well remove the whitespace code blocks and finally close this issue.
I'll do a PR for that.
